### PR TITLE
Add luminosity functions to colibre

### PR DESCRIPTION
**Pull request by MatthieuSchaller**
Also update the data submodule to the latest version.

### CODE REVIEW

File: colibre/registration.py [hunk 1]
```diff
@@ -901,3 +927,4 @@ def register_stellar_mass_scatter(self, catalogue, stellar_mass_scatter_amplitud
 register_star_Mg_and_O_to_Fe(self, catalogue, aperture_sizes_30_50_100_kpc)
 register_gas_fraction(self, catalogue)
 register_stellar_mass_scatter(self, catalogue, stellar_mass_scatter_amplitude)
+register_star_magnitudes(self, catalogue, aperture_sizes)
```
Comment (EvgeniiChaikin) [inline colibre/registration.py:930]:
```suggestion
register_star_magnitudes(self, catalogue, aperture_sizes_30_50_100_kpc)
```

**Review by EvgeniiChaikin — COMMENTED** [review]
Thanks @MatthieuSchaller 

One comment - don't we want to use the 50 kpc apertures, like in the other plots?

File: colibre/registration.py (hunk 1 above)
Comment (MatthieuSchaller) [reply colibre/registration.py:930]: Yes, good point. 30kpc was the eagle choice. I should have changed it.

Can you make the change here? I am off laptop today.

Comment (EvgeniiChaikin) [reply colibre/registration.py]: Sure, done

**Comment by MatthieuSchaller** [comment]
Thanks!

**Pull request merged**